plugins/file/index.js

```diff
@@ -69,6 +70,8 @@ module.exports = {
       },
       handler: async function(request, h) {
         const file = request.payload.file;
+        const fullPath = request.payload.fullPath;
```

Comment:
Lets rename `fullPath` to `fileKey`. This name was already used to reference the full key before in the code